File: WebUI/UI/src/app/datas/models/base/question.ts
export class Question {

    constructor(questionTypeId: number, formId: number,  title: string, description: string, questionOrder: number) {
        this.questionTypeId = questionTypeId;
        this.formId = formId;
        this.title = title; 
        this.description = description;
        this.questionOrder = questionOrder;    
    }

    id: number;
    questionTypeId: number;
    formId: number;
    title: string;
    description: string;
    questionOrder: number;
}

File: Entities/Base/Question.cs
using Core.Entities;

namespace Entities.Base
{
    public class Question:IEntity
    {
        public int Id { get; set; }
       public int QuestionTypeId { get; set; } 
       public int FormId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int QuestionOrder { get; set; }

        //public QuestionType QuestionType { get; set; }
    }
}
